meshroom/core/desc.py

```diff
@@ -133,6 +145,24 @@ def validateValue(self, value):
 
         return value
 
+    def validateValueType(self):
+        """ Check the default value's and range's (if available) type of every attribute contained in the group
+        (including nested attributes).
+        Returns an empty string if all the attributes' types are valid, or concatenates the names of the attributes in
+        the group with invalid types.
+        """
+        invalidParams = []
+        for attr in self.groupDesc:
+            name = attr.validateValueType()
+            if name:
+                invalidParams.append(name)
+        if invalidParams:
+            # In group "group", if parameters "x" and "y" (with "y" in nested group "subgroup") are invalid, the
+            # returned string will be: "group:x, group:subgroup:y"
```

Comment:
I'm not sure I understand this comment. Does it address only the comment itself, or the way nested attributes are displayed in the string?

If it is the former, then no, the comment is correct. If we take for example the "x" attribute of "principalPoint" (which is itself in the "intrinsic" group) in the CameraInit node, it would be displayed as "intrinsic:principalPoint:x" (as opposed to, let's say, "focalLength", which is directly in "intrinsic" and which would be displayed as "intrinsic:focalLength").

If it's the latter, I can update it but I think it might be easier to fix the erroneous attributes if the precise "position" of the attributes is given.

Reply:
just misread